src/main/java/org/myrobotlab/service/MouthControl.java

```diff
@@ -114,25 +165,98 @@ public synchronized void onEndSpeaking(String utterance) {
     jaw.moveTo((double) mouthClosedPos);
   }
 
-  public void setdelays(Integer d1, Integer d2, Integer d3) {
+  public void setDelays(Integer d1, Integer d2, Integer d3) {
     delaytime = d1;
     delaytimestop = d2;
     delaytimeletter = d3;
   }
 
-  public void setmouth(Integer closed, Integer opened) {
-    // jaw.setMinMax(closed, opened);
+  
+  @Deprecated /* use setDelays */
+  public void setdelays(Integer d1, Integer d2, Integer d3) {
+    setDelays(d1, d2, d3);
+  }
+
+  public void setMouth(Integer closed, Integer opened) {
     mouthClosedPos = closed;
     mouthOpenedPos = opened;
+  }
+  
+  @Deprecated /* use setMouth */
+  public void setmouth(Integer closed, Integer opened) {
+    setMouth(closed, opened);
+  }
 
-    // jaw.setMinMax(closed, opened);
-    // if (closed < opened) {
-    // jaw.map(closed, opened, closed, opened);
-    // } else {
-    // jaw.map(opened, closed, opened, closed);
-    // }
+  
+  @Override
+  public void onInterfaceRegistered(String serviceName, String interfaceName) {
+    if (ServoControl.class.toString().equals(interfaceName)) {
+      servoServices.add(serviceName);

```

Comment:
we shouldn't be tracking the list of servo services in this service.  that list should come from runtime..

Reply:
yeah - this is no longer applicable ... I'll update soon

Reply:
method is gone - all registering for interface now use normalized runtime data and appropriate callbacks